src/cli-options.ts

```diff
@@ -194,6 +194,9 @@ export const getOptions = (): Result<Options> => {
     if (str === 'quiet') {
       return {ok: true, value: new QuietLogger(packageRoot)};
     }
+    if (str === 'quiet-ci') {
+      return {ok: true, value: new QuietCiLogger(packageRoot)};
+    }
```

Comment:
This can be addressed later when changing default to 'quiet' as well, but wdyt about defaulting to 'quiet-ci' based on `process.env['CI']`? We have precedence for that for the cache option here https://github.com/google/wireit/blob/9a78f7ff363fb9653afb9ca2177deaaa9581c2fb/src/cli-options.ts#L126-L142

Reply:
Yes, agree. When we default to quiet we should default to quiet-ci in ci.